src/utility/index.js

```diff
@@ -220,9 +340,9 @@ const isValidVariablesShape = variables =>
 /**
  * Return choice with command directive stripped
  *
- * @param string documentedChoice     - documented choice to be undocumented
+ * @param {string} documentedChoice     - documented choice to be undocumented
  *
- * @returns string undocumentedChoice - choice with documented command directive stripped
+ * @returns {string} undocumentedChoice - choice with documented command directive stripped
  */
 const getUndocumentedChoice = documentedChoice => {
   if (documentedChoice.indexOf(chalk.blue.bold('→')) !== -1) {
```

Comment:
All of these documentation choices could be put into an array/map and then it would just be a loop to check for and remove them.